src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetPlatforms.props

```diff
@@ -0,0 +1,21 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.SupportedTargetPlatforms.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project>
+  <ItemGroup>
+    <SupportedPlatform Include="Android" />
```

Comment:
I would add a comment here explaining what these items are for, preferably linking to a spec.